common/main/af.xml

```diff
@@ -9455,157 +9455,157 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 	</typographicNames>
 	<personNames>
 		<nameOrderLocales order="givenFirst">und af</nameOrderLocales>
-		<nameOrderLocales order="surnameFirst">ja ko vi yue zh</nameOrderLocales>
-		<foreignSpaceReplacement xml:space="preserve"> </foreignSpaceReplacement>
+		<nameOrderLocales order="surnameFirst">↑↑↑</nameOrderLocales>
+		<foreignSpaceReplacement xml:space="preserve">↑↑↑</foreignSpaceReplacement>
```

Comment:
this means inheritance from root? root does have "ja ko vi yue zh"

this is still a puzzle though since replaceBaileyWithInheritanceMarker explicitly excludes inheritance from root

Reply:
It is not a problem to inherit from root IF the inheritance is not lateral (and even there sometimes not). We have some more to do, but the first step is bringing in the data from VXML without stomping on the uninvolved data.